test/masq.test.js

```diff
@@ -214,10 +210,21 @@ test('should exchange key and authorize local key if challenge matches', async (
   masq.exchangeDataHyperdbKeys(appName)
 })
 
+/**
+ * Return when hyperDb instance is ready
+ * @param {Object} db - The hyperDb instance
+ */
+const dbReady = (db) => {
```

Comment:
Please move this util function at the top of the test file

Reply:
removed as it was not used